src/web/components/Title/Title.test.js

```diff
@@ -6,15 +6,15 @@ describe('Title', () => {
   let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(<Title>Hello World</Title>);
+    wrapper = shallow(<Title>Hello World</Title>).shallow();
```

Comment:
Same here, I don't get the point

Reply:
So far, this is the only solution I have found to test a component that uses an HOC. I see that there is a problem because if I use more than one HOC, I will have to use more than one shallow for each level of rendering depth I want. 

So, do you have a method to test only the component that uses HOC?

Reply:
Update : I do my test without the HOC.